modules/db_parameter_group/data.tf

```diff
@@ -0,0 +1,3 @@
+locals {
```

Comment:
Please put `locals` in `main.tf`. No need for `data.tf`.

Reply:
Done